src/cryptsetup/cryptsetup.c

```diff
@@ -851,20 +851,25 @@ static int acquire_pins_from_env_variable(char ***ret_pins) {
 }
 #endif
 
-static int attach_luks2_by_fido2_via_plugin(
+static int crypt_activate_by_token_pin_ask_password(
                 struct crypt_device *cd,
                 const char *name,
+                const char *type,
                 usec_t until,
                 bool headless,
                 void *usrptr,
-                uint32_t activation_flags) {
+                uint32_t activation_flags,
+                const char *message,
+                const char *key_name,
+                const char *credential_name
+                ) {
```

Comment:
please merge this line into the last. i.e. we generally put the closing `) {` on the same line as the last argument

Reply:
Sure, done :)